packages/vlc.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,40 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $toolName = 'VLC'
```

Comment:
This could just use the existing VLC package at https://community.chocolatey.org/packages/vlc
I'm not sure we need it in FLARE VM though. How do you see this being used during RE?

Reply:
Yes we could. It is currently under vlc.fireye, I was porting the utilities first.. But no need to maintain two.